examples/auctions/blind_auction.vy

```diff
@@ -24,16 +19,17 @@ highestBid: public(wei_value)
 highestBidder: public(address)
 
 # State of the bids
-bids: map(address, Bid[128])
-bidCounts: map(address, int128)
+bids: map(bytes32, wei_value) # Indexed by blinded bid
```

Comment:
Might want to double map `address => bidHash => bidBlindAmt` for extra protection

Reply:
You can do this instead of having this in the hashing portion of the blinded bid

Reply:
As currently in the PR there is an additional feature that the bidder is hidden, and it is more general. It's intentional.

Adding a map adds an additional restriction that bids can only be submitted by the person that will unseal them.

Reply:
Fair